src/go/tools.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build tools
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/golang/mock/mockgen"
```

Comment:
Not sure what this file is doing generally `src/go/tools.go` - can you briefly explain for me? Do these tree modules require initialization for general use, and so you group them here to simplify import by other? or..

Reply:
https://github.com/golang/go/issues/25922

this is the convention for getting `go mod tidy` to work when you have dev tools that are not used by code.